source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,577 +3,765 @@
+
+.. code-block:: bash
+
+   sudo cp /usr/share/defaults/ister/ister.json relase-image-config.json
+
+For reference, you can inspect the `Clear Linux ister configuration file`_
+used for releases.
+
+Edit the configuration file to include all bundles you want *preinstalled* in
+the image. Users can install the bundles in the mix not included in the
+configuration file with the following command:
+
+.. code-block:: bash
+
+   sudo swupd bundle add
+
+Keeping the list of bundles in the configuration file small allows for a
+smaller image size. For the minimal base image, the list is:
+
+.. code-block:: console
+
+   "Bundles": ["os-core", "os-core-update", "kernel-native"]
+
+Next, set the `Version` field to the mix version content mixer should use to
+build image. `ister` allows you to build an image from any mix version you
+have built, not just the current one. In our example so far, `Version` is set
+to 10.
+
+With the `ister` tool configured, build the image with with the following command:
+
+.. code-block:: bash
+
+   sudo mixer build image --format 1
+
+This command outputs an image bootable as a virtual machine and which can be
+installed on bare metal.
+
+Mixer automatically looks for the :file:`release-image-config.json` file but
+you can freely choose the filename. To use a different name, simply pass the
+:option:`--template` flag when creating your image, for example:
+
+.. code-block:: bash
+
+   sudo mixer build image --format 1 --template path/to/file.config
+
+By default, `ister` uses the format version of the build machine it runs on.
+Therefore, if the format you are building differs from the format of the |CL|
+OS you are building on, you must use the :option:`--format <FORMAT_NUMBER>`
+flag. Find the current format version of your OS with the following command:
+
+.. code-block:: bash
+
+   sudo cat /usr/share/defaults/swupd/format
+
+Update the next mix version information
+***************************************
+
+Increment the mix version number for the next mix with the following command:
+
+.. code-block:: bash
+
+   sudo mixer versions update
+
+This command automatically updates the mix version stored in the
+:file:`mixversion` file incrementing it by 10. To increment by a different
+amount, use the :option:`--increment` flag, for example:
+
+.. code-block:: bash
+
+   sudo mixer versions update --increment 100
+
+Alternatively, to set the mix version to a specific value, use the
+:option:`--mix-version` flag, for example:
+
+.. code-block:: bash
+
+   sudo mixer versions update --mix-version 200
+
+The :command:`mixer versions update` command does not allow you to set the mix
+version to a value lower than its current value. The mix version is expected
+to always increase, even if the new mix is undoing an earlier change.
+
+If you have been tracking your workspace with Git, you can restore the mix to
+an earlier state but be careful of "rewriting history" if you are publishing
+the mix content to users already.
```

Comment:
You removed the comma before "but". Without it, this is a run-on sentence.

Requested edit: "state but" -> "state, but"

Reply:
Done by Evan